### PR TITLE
Cb test fail

### PR DESCRIPTION
**Pull request by mayou36**
Fixes #

## Proposed Changes

  -
  -
  -

## Tests added

  -
  -
  -
  
## Checklist

 - [x] change approved
 - [x] implementation finished
 - [x] correct namespace imported
 - [x] tests added
 - [x] CHANGELOG updated

### CODE REVIEW

File: tests/test_pdf_cb.py [hunk 1]
```diff
@@ -91,7 +91,7 @@ def test_cb_dcb():
     assert not any(np.isnan(probsr))
 
     probsl_scipy = crystalball.pdf(x, beta=alphal, m=nl, loc=mu, scale=sigma)
-    probsr_scipy = crystalball.pdf(-x, beta=alphar, m=nr, loc=mu, scale=sigma)
+    probsr_scipy = crystalball.pdf(-x + 2 * mu, beta=alphar, m=nr, loc=mu, scale=sigma)
```
Comment (marinang) [inline tests/test_pdf_cb.py]: Is this dark magic :D ?

Comment (mayou36) [reply tests/test_pdf_cb.py]: Haha kinda: we want to mirror it at the mu. x is not at mu, we mirror it (now at -mu), shift it back.

Simple math thing, mirroring at point mu

**Comment by mayou36** [comment]
Sorry for already merged, did not expect you to have time now ;) Let me know if you have another comment, there have been a few changes (see updated changelog)

File: tests/test_pdf_cb.py (hunk 1 above)
Comment (marinang) [reply tests/test_pdf_cb.py]: Oh yeah cause mu is not equal to zero anymore.

Comment (mayou36) [reply tests/test_pdf_cb.py]: yep, just breaking some symmetries to see things still work

**Comment by marinang** [comment]
@mayou36 It is fine no worries, I was just curious about this PR since I also was somehow affected by the CB a while ago.

**Comment by mayou36** [comment]
Ah yep, solutions: use the new option "use_minuit_grad" in Minuit to use the numerical gradient, then things work out better

**Pull request merged**